docs/topics/auth/passwords.txt

```diff
@@ -211,6 +215,34 @@ follows:
 
 .. _password-upgrades:
 
+Scrypt
+~~~~~~
+
+The scrypt hashing algorithm is similar to PBKDF2 and bcrypt in utilizing
+a set number of iterations to slow down brute-force attacks. However, because
+PBKDF2 and bcrypt do not require very much memory, attackers with sufficient 
+resources can launch large-scale parallel attacks in order to speed up the 
+attacking process. The scrypt algorithm is specifically designed to use large
+amounts of memory compared to other password-based KDFs in order to limit
+the amount of parallelism an attacker can use.
+
+Scrypt has three attributes that can be customized:
+
+#. ``work_factor`` controls the number of iterations within the hash.
+#. ``block_size`` controls the block size, which fine-tunes sequential memory
+   read size and performance.
+#. ``parallelism`` controls how many threads will run in parallel.
+
+The default values for the respective attributes are:
+
+#. ``work_factor`` has a default value of 16384.
+#. ``block_size`` has a default value of 8.
+#. ``parallelism`` has a default value of 1.
+
+.. note::
+
+    Scrypt is only available in the Python standard library for Python 3.6 and up.
```

Comment:
This isn't necessary as Django doesn't support versions older than 3.6. I see that Python's documentation for `hashlib.script` notes a requirement of OpenSSL 1.1+. It would be worth looking at common systems today to see if that's something that might be worth noting.

Reply:
Removed Python 3.6 note and added note regarding OpenSSL 1.1+.